pkg/resource/cluster/hooks.go

```diff
@@ -262,3 +263,123 @@ func (rm *resourceManager) newMemoryDBClusterUploadPayload(
 
 	return res
 }
+
+// getTags gets tags from given ParameterGroup.
+func (rm *resourceManager) getTags(
+	ctx context.Context,
+	resourceARN string,
+) ([]*svcapitypes.Tag, error) {
+	resp, err := rm.sdkapi.ListTagsWithContext(
+		ctx,
+		&svcsdk.ListTagsInput{
+			ResourceArn: &resourceARN,
+		},
+	)
+	rm.metrics.RecordAPICall("GET", "ListTags", err)
+	if err != nil {
+		return nil, err
+	}
+	tags := make([]*svcapitypes.Tag, 0, len(resp.TagList))
+	for _, tag := range resp.TagList {
+		tags = append(tags, &svcapitypes.Tag{
+			Key:   tag.Key,
+			Value: tag.Value,
+		})
+	}
+	return tags, nil
+}
+
+// updateTags updates tags of given ParameterGroup to desired tags.
+func (rm *resourceManager) updateTags(
+	ctx context.Context,
+	desired *resource,
+	latest *resource,
+) (err error) {
+	rlog := ackrtlog.FromContext(ctx)
+	exit := rlog.Trace("rm.syncTags")
```

Comment:
this should match the function name; so either rename the function `syncTags` or change this to `rm.updateTags`

Reply:
You're right. Forgot changing it.